themes/default/content/blog/the-present-and-future-of-ai-and-iac/index.md

```diff
@@ -0,0 +1,111 @@
+
+One of the most striking things about modern AI-based applications is that they are significantly more cloud-native than previous generations. This is true for several reasons:
+
+* Modern AI technology and services have been built around cloud-native assumptions
+* AI-based applications frequently have especially bursty compute needs
+* AI applications and the space overall is scaling rapidly, requiring elasticity and low operational overhead
+* New managed services, like vector datastores, managed model serving APIs, managed data and training pipelines
+
+This cloud-native quality of AI-based applications makes Infrastructure as Code even more important and valuable for AI applications than in other, more traditional applications.
+
+The services to support building AI applications have all been built over the last two years around the assumption of the cloud. GPUs are dominantly accessed through cloud services. LLMs are invoked via managed services from OpenAI or via managed hosting of open-source models via SageMaker, Huggingface, or others. Retrieval Augmented Generation (RAG) is powered by dozens of managed database offerings, from dedicated vector databases like Pinecone, to vector extensions in Amazon Aurora PostgreSQL and more. Developers coming to AI assume they will stitch these services into rich application offerings.
+
+At Pulumi, we've seen [dozens of AI-focused companies](/solutions/ai/) adopting and using Pulumi to manage their infrastructure, motivated by a combination of these AI-oriented infrastructure needs, along with the ease of using Python (already heavily used within the AI space), and TypeScript (popular due to a wide variety of AI-oriented front end frameworks). Pulumi also exposes the ability to manage many of these services directly, provisioning and managing the lifecycle of AI services across your infrastructure through support for things like [Azure OpenAI Service](/registry/packages/azure-native/api-docs/cognitiveservices/deployment/), [Pinecone Indexes](/registry/packages/pinecone/api-docs/pineconeindex/), [Amazon SageMaker Endpoints](/registry/packages/aws/api-docs/sagemaker/endpoint/), and much more.
+
+As these applications move from prototypes to production, the need for robust and scalable infrastructure to deploy these applications in the major cloud platforms is essential, and IaC offers the best way to manage this at scale. For example, the production-grade infrastructure for [deploying AI applications leveraging Pinecone’s vector database at scale](https://www.pulumi.com/blog/pinecone-serverless/) highlights an AWS architecture for running an AI application in a production environment.
```

Comment:
We should cross-reference this new blog post on Langchain https://www.pulumi.com/blog/easy-ai-apps-with-langserve-and-pulumi/

We may also want to drop in this cool arch diagram https://www.pulumi.com/blog/easy-ai-apps-with-langserve-and-pulumi/architecture.png